### DISCUSSION TITLE
Geomap: variable maps names and colors

### DISCUSSION THREAD

**Discussion by zufolo441** [grafana/grafana#46400] [Dashboards, Panels & Visualizations]
Hi, I have to represent statistical data in territories, by colouring areas.
Here an example that I made with Apache Superset for an italian region.
Questions: is there a way to use variables in file name of geojson files?
is there a way to set colors based on sql data? 

Thanks!
![Screenshot_20220310_092650](https://user-images.githubusercontent.com/45826835/157619855-9196edf9-a020-47aa-a082-7fad4983c437.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
